### PR TITLE
Docs and meta information update for release

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -292,6 +292,7 @@ def main():
             "Programming Language :: Python :: 3.8",
             "Programming Language :: Python :: 3.9",
             "Programming Language :: Python :: 3.10",
+            "Programming Language :: Python :: 3.11",
```
Comment (alalek) [inline setup.py:296]: drop line 290 with python 3.6?

File: README.md [hunk 2]
```diff
@@ -216,15 +216,16 @@ The default ``manylinux2014`` images have been extended with some OpenCV depende
...
 - 3.10
+- 3.11
 
 ### Backward compatibility
 
 Starting from 4.2.0 and 3.4.9 builds the macOS Travis build environment was updated to XCode 9.4. The change effectively dropped support for older than 10.13 macOS versions.
 
 Starting from 4.3.0 and 3.4.10 builds the Linux build environment was updated from `manylinux1` to `manylinux2014`. This dropped support for old Linux distributions.
 
+Starting from version 4.7.0 the Mac OS Github Actions build environment was update to version 11. Mac OS 10.x support depricated. See https://github.com/actions/runner-images/issues/5583
```
Comment (alalek) [inline README.md]: Github -> GitHub

File: setup.py (hunk 1 above)
Comment (asmorkalov) [reply setup.py:296]: The package is build from sources for python 3.6. I do not see strong reason to drop it for now.

**Comment by asmorkalov** [comment]
Looks like the same issue like https://stackoverflow.com/questions/65888506/error-wheel-whl-is-not-a-supported-wheel-on-this-platform

File: README.md [hunk 3]
```diff
@@ -216,15 +216,16 @@ The default ``manylinux2014`` images have been extended with some OpenCV depende
...
 - 3.10
+- 3.11
 
 ### Backward compatibility
 
 Starting from 4.2.0 and 3.4.9 builds the macOS Travis build environment was updated to XCode 9.4. The change effectively dropped support for older than 10.13 macOS versions.
 
 Starting from 4.3.0 and 3.4.10 builds the Linux build environment was updated from `manylinux1` to `manylinux2014`. This dropped support for old Linux distributions.
 
+Starting from version 4.7.0 the Mac OS GitHub Actions build environment was update to version 11. Mac OS 10.x support depricated. See https://github.com/actions/runner-images/issues/5583
```
Comment (ilya-lavrenov) [inline README.md:231]: depricated => deprecated

**Pull request merged**